### ISSUE TITLE
Add instructions to give service account permission to write on user bucket

### ISSUE THREAD

**Issue by ivamluz** [dviveiros/paul-octopus-java#4]
By default, buckets only have permission to the project owners and to each user, so it would be good to add some note on the README, so users remember to give the proper access to the service account the app runs on behalf of.

**Comment by dviveiros** [comment]
I'm using the authentication-default, right? Shouldn't it consider the user credentials?
I will keep this one open and check with the team here is an extra step is really required.

**Comment by ivamluz** [comment]
Yes, if you use authentication-default it should work, but, since anyone has access to the machines, that could be considered a security risk (a person could ssh into your instance and use your credential on behalf of you).
